### ISSUE TITLE
[Revise subclass] MONDO:0025481 zoonoses

### ISSUE THREAD

**Issue by paolaroncaglia** [monarch-initiative/mondo#1467]
**MONDO:0025481 zoonoses**

"Diseases of non-human animals that may be transmitted to humans or may be transmitted from humans to non-human animals."

Currently Subclass of:
    non-human animal disease
    parasitic infection
    viral infectious disease

[Wikipedia](https://en.wikipedia.org/wiki/Zoonosis) says: "A zoonosis (plural zoonoses, or zoonotic diseases) is an infectious disease caused by a pathogen (an infectious agent, including bacteria, viruses, parasites, prions, etc) that has jumped from non-human animals (usually vertebrates) to humans."

Would you please consider:
- [x] Removing the superclass 'non-human animal disease' (as zoonoses may affect humans as well as animals)
- [x] Removing the superclass 'viral infectious disease' (in fact, atm the only child of 'zoonoses' is 'zoonotoc bacterial infection' ;-) )
- [x] Possibly removing the superclass 'parasitic infection' (zoonoses may be caused by viruses or bacteria, which are not commonly thought of as parasites even though they are (compare the children of MONDO:0005135 'parasitic infection', none of which is due to bacteria or viruses, with the [Wikipedia overview of parasites' taxonomic range](https://en.wikipedia.org/wiki/Parasitism#Taxonomic_range))

Basically I suggest to have 'infectious disease' be the only parent. Also please consider

- [x] Editing the label to 'zoonosis' (keeping "zoonoses" as synonym, and possibly adding "zoonotic disease" and "zoonotoc infection" as synonyms)
- [ ] Expanding the definition 
- [x] Adding x-ref Wikipedia:Zoonosis

Thanks!

(Note for self: this affects EFO.)

**Comment by paolaroncaglia** [comment]
Hi again @nicolevasilevsky ,
This request is also pending PR, would there be any objection to pushing it please? Many thanks.
